src/components/InputGroup.tsx

```diff
@@ -1,14 +1,22 @@
-import ErrorMessage from './ErrorMessage.jsx'
+import React from 'react'
+import ErrorMessage from './ErrorMessage.js'
 import S from 'styled-components'
 import { useState } from 'react'
+import { InputsProps } from './SplittedInput.js';
```

Comment:
remove.js

Reply:
@itmilos  I do not know why is that happening. The app is constantly injecting .jsx imports